backends/qualcomm/utils/utils.py

```diff
@@ -299,6 +299,7 @@ def get_decomp_table() -> Dict[torch._ops.OperatorBase, Callable]:
 def _transform(
     edge_program: ExportedProgram, custom_pass_config: FrozenSet[str] = frozenset()
 ) -> ExportedProgram:
+    custom_pass_config = custom_pass_config or {}
```

Comment:
Since `custom_pass_config` is now default constructed, could you remove this line?

Reply:
It has already been removed, thank you.